#25 - Sponsor page use yml

jcklie opened the pull request:
- Use new real data for sponsor page
- Add images

Needs https://github.com/acl-org/acl-2020-virtual-conference-sitedata/pull/10

I did not change the layout yet.

jcklie commented:
The session timestamps are not localizing yet, I tried the same solution as on the https://github.com/acl-org/acl-2020-virtual-conference/blob/master/templates/poster.html but I did not get it working yet.

GuyAglionby commented:
The [regex](https://github.com/acl-org/acl-2020-virtual-conference/blob/a0982cbd6930deedff415365658fcc64def01ccc/static/js/time-extend.js#L2) to match times is fussy and wants it to be surrounded by parentheses. This should probably be relaxed - the poster page things can be easily changed to allow this ([here](https://github.com/acl-org/acl-2020-virtual-conference/blob/a0982cbd6930deedff415365658fcc64def01ccc/templates/poster.html#L73), [here](https://github.com/acl-org/acl-2020-virtual-conference/blob/a0982cbd6930deedff415365658fcc64def01ccc/main.py#L61))

hao-fang commented on static/images/sponsors/foo.txt:
```diff
@@ -0,0 +1,18 @@
+https://acl2020.org/assets/images/logos/Bloomberg-logo-2019.png
```
do we need this `foo.txt` file?

jcklie commented:
I fixed the times, update the submodule and deleted the bogus files. 

![image](https://user-images.githubusercontent.com/934600/84881966-deeadd00-b08e-11ea-93db-d4cbe10916fd.png)

Are we gonna keep this font and blue header color? For me, it does not look ACL like.

jcklie commented:
Can you help me with the submodule and why there is a conflict? I am not too familiar with submodules.

hao-fang commented:
> Can you help me with the submodule and why there is a conflict? I am not too familiar with submodules.

Since the PRs on the `sitedata_acl2020` are merged, you can do
```bash
cd sitedata_acl2020
git pull
cd ..
git commit -am "bump sitedata_acl2020 version"
```
This should work. But let me know if doesn't -- we can chat on RocketChat.
